### DISCUSSION TITLE
Completely override lualine

### DISCUSSION THREAD

**Discussion by rubiin** [LazyVim/LazyVim#3414] [General]
I have a custom lualine setup. SInce lualine is integrated on core, is there is a way to override everything and just use my settings.

**Comment by dpetka2001** [comment]
You could do something like
```lua
return {
  {
    "lualine.nvim",
    opts = function()
	  local opts = {}
      return opts
    end,
  },
}
```
Put whatever you want inside the `local opts` table. This should work.

**Reply by rubiin** [reply]
lemme try

**Reply by rubiin** [reply]
I guess i would have to add it before my spec right?

**Reply by dpetka2001** [reply]
Just put that in a file in `/lua/plugins/` directory and put your own settings for lualine inside `local opts` table.
